components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1418,17 +1452,28 @@
 <mesh_file hgrid="ne0np4_enax4v1">atm/cam/inic/homme/enax4v1.g</mesh_file>
 <mesh_file hgrid="ne0np4_twpx4v1" >atm/cam/inic/homme/twpx4v1.g</mesh_file>
 
+
+<!-- viscosity  -->
+
 <nu_top> 2.5e5 </nu_top>
-<nu_top hgrid="ne240np4"> 1.0e5 </nu_top>
+<nu_top dyn_target="preqx" hgrid="ne240np4"> 1.0e5 </nu_top>
+<nu_top dyn_target="theta-l" hgrid="ne120np4"> 1e5 </nu_top>
+<nu_top dyn_target="theta-l" hgrid="ne240np4"> 4.3e4 </nu_top>
+<nu_top dyn_target="theta-l" hgrid="ne256np4"> 4e4 </nu_top>
+<nu_top dyn_target="theta-l" hgrid="ne512np4"> 2e4 </nu_top>
+<nu_top dyn_target="theta-l" hgrid="ne1024np4"> 1e4 </nu_top>
```

Comment:
Is if fair to say these nu_top coefs are still wip?

Reply:
yes, these are the ones that work with hsv_tom=1.   since we dont know the correct value for high-res, i thought we should start with the most efficient.